app/modules.go

```diff
@@ -220,8 +221,8 @@ NOTE: capability module's beginblocker must come before any modules using capabi
 
 func orderBeginBlockers() []string {
 	return []string{
-		upgradetypes.ModuleName,
 		capabilitytypes.ModuleName,
+		upgradetypes.ModuleName,
```

Comment:
this should fix the issue with calling ica txs from upgrade handlers!

Reply:
cool